Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "onecode"
+version = "0.1.0"
+authors = ["humb1t"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+serde = { version = "1.0", features = ["derive"] }
```

Comment:
This is okay for an MVP, but we should try to limit dependencies to the bare minimum. Working with financial institutions, for some clients we have to actually explain and get permission for every library used and this can take time. For Iroha2 I would like to try to write a lot of it ourselves because: 1) this isn't magic :), and 2) it gives us more control over details of the code, which can be important when we need to guarantee there are no problems or to improve performance.